packages/storage/src/file.js

```diff
@@ -269,6 +269,12 @@ function File(bucket, name, options) {
 
 util.inherits(File, common.ServiceObject);
 
+/*! Developer Documentation
+ *
+ * @param {object=} options - Configuration object.
```

Comment:
Just to clarify, we're not publicly documented options because the user will never manually make the subsequent requests for large files?